Source/WTF/wtf/PlatformHave.h

```diff
@@ -1833,6 +1833,10 @@
 #define HAVE_SUPPORT_HDR_DISPLAY 1
 #endif
 
+#if PLATFORM(MAC)
+#define ENABLE_FULLSCREEN_VIDEO_PREVENT_HDR_SUPPRESSION 1
```

Comment:
`ENABLE_` should go in `PlatformEnable.h`. Here it would be something like:

`HAVE_FULLSCREEN_VIDEO_PREVENTS_HDR_SUPPRESSION`